helm/reana-stable/values.yaml

```diff
@@ -0,0 +1,123 @@
+# Default values for reana.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# TODO Document this shuld be set for third party integrations and for
+# production deployments due to security issues.
+# reana_url: reana.cern.ch
+
+# Storage configuration
+# reana_url: reana.cern.ch
+debug:
+  enabled: false
+ui:
+  enabled: false
+eos:
+  enabled: false
+# CERN specific for now
+cephfs:
+  enabled: false
+  cephfs_volume_size: 200
+  access_modes: ReadWriteMany
+  provisioner: manila-provisioner
+  type: "Geneva CephFS Testing"
+  os_necret_name: os-trustee
+  os_necret_namespace: kube-system
+  cephfs_os_share_id: <cephfs-share-id>
+  cephfs_os_share_access_id: <cephfs-share-access-id>
+secrets:
+  database:
+    user: reana
+    # pasword: <my-db-pass>
+  gitlab:
+    REANA_GITLAB_OAUTH_APP_ID: reana
+    # REANA_GITLAB_OAUTH_APP_SECRET: <gitlab_oauth_app_secret>
+    REANA_GITLAB_HOST: gitlab.cern.ch
+  cern:
+    sso:
+      CERN_CONSUMER_KEY: reana
+      # CERN_CONSUMER_SECRET: <cern_consumer_secret
+
+# External database service configuration
+db_env_config:
+  REANA_DB_NAME: "reana"
+  REANA_DB_HOST: "reana-db"
+  REANA_DB_PORT: "5432"
+  # There are two more environment variables that should be set in order
+  # to connect to a database:
+  # REANA_DB_USERNAME: containing the database user name.
+  # REANA_DB_PASSWORD: containing the password for the user previously set.
+  # Both environment variables should be set inside a Kubernetes secret:
+  # `reana-db-secrets`
+
+# Filesystem related configuration variables
+volume_paths:
+  root_path: "/var/reana"
+  shared_volume_path: "/var/reana"
+  db_persistence_path: "/var/reana/db"
+
+# REANA components configuration
+components:
+  reana_server:
+    imagePullPolicy: IfNotPresent
+    image: reanahub/reana-server
+    tag: 0.6.0
+    environment: []
+  reana_workflow_controller:
+    imagePullPolicy: IfNotPresent
+    image: reanahub/reana-workflow-controller
+    tag: 0.6.0
+    environment:
+      SHARED_VOLUME_PATH: /var/reana
+  reana_workflow_engine_cwl:
+    image: reanahub/reana-workflow-engine-cwl
+    tag: 0.6.0
+  reana_workflow_engine_yadage:
+    image: reanahub/reana-workflow-engine-yadage
+    tag: 0.6.0
+  reana_workflow_engine_serial:
+    image: reanahub/reana-workflow-engine-serial
+    tag: 0.6.0
+  reana_job_controller:
+    image: reanahub/reana-job-controller
+    tag: 0.6.0
+  reana_message_broker:
+    imagePullPolicy: IfNotPresent
+    image: reanahub/reana-message-broker
+    tag: 0.6.0
+  reana_ui:
+    imagePullPolicy: IfNotPresent
+    image: reanahub/reana-ui
+    tag: 0.6.0
```

Comment:
Let's use `latest` instead of `0.6.0` since the current setup is developer-oriented; the image tags will be replaced when publishing versioned-charts later.